shopfloor/tests/test_cluster_picking_scan_line.py

```diff
@@ -72,10 +102,20 @@ def test_scan_line_serial_ok(self):
         self._scan_line_ok(line, line.lot_id.name)
 
     def test_scan_line_error_product_tracked(self):
-        """Scan a product tracked by lot, must scan the lot"""
+        """Scan a product tracked by lot, must scan the lot.
+
+        If for the same product there is multiple lot in the location.
+        The user need to scan the lot.
```

Comment:
```suggestion
        If for the same product there is multiple lots in the location,
        the user must scan the lot.
```

Reply:
But if the only product available in the operation processed is in only one lot among reserved move lines, and in a second lot in another operation for the same location (but we don't care about), why should we block the scan? :thinking: